frontend/package.json

```diff
@@ -68,16 +68,21 @@
     "@trivago/prettier-plugin-sort-imports": "^4.3.0",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
+    "eslint-import-resolver-typescript": "^3.7.0",
```

Comment:
needed for the import checking with Typescript, hardest challenge of this entire POC :)